docs/Migration.md

```diff
@@ -0,0 +1,33 @@
+# Migrate an existing Android app to AEP Android SDK 2.0
+
+This document describes how existing Android SDK 1.x customers can upgrade to the Android SDK 2.0 without introducing any breaking changes. We have made upgrading seamless, which will ensure that APIs between SDK 1.x and 2.0 are compatible.
+
+## Upgrading
+
+### Gradle
+
+To upgrade, update the Gradle dependency references to `Android SDK 2.0` in the Android app's `build.gradle`:
+
+```diff
+- implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+
++ implementation 'com.adobe.marketing.mobile:core:2.0.0'
```

Comment:
Can you also add a note telling them to update gradle dependencies to latest available versions